core/templates/dev/head/pages/admin/admin_navbar_directive.html

```diff
@@ -128,15 +134,16 @@
             <a class="oppia-navbar-tab protractor-test-admin-config-tab"
                ng-click="showTab(ADMIN_TAB_URLS.CONFIG)"
                ng-href="<[::ADMIN_TAB_URLS.CONFIG]>"
+               ng-class="{active: isConfigTabOpen()}"
                uib-tooltip="Config"
                tooltip-placement="bottom">
               Config
             </a>
           </li>
           <li class="oppia-clickable-navbar-element pull-right">
             <a class="oppia-navbar-tab"
-               ng-click="showTab(ADMIN_TAB_URLS.JOBS)"
```

Comment:
Interesting, why delete this?

If unintentional, please make sure to always double-check the "Files Changed" tab after you push.

Reply:
We don't need this since everything is tracked by $locationChangeSuccess event

Reply:
But why do it just for JOBS and not for the others?

Reply:
Sorry, seems like I missed them. Should I delete them and submit another commit to this branch?

Reply:
Yep, please go ahead. It's totally fine to make multiple commits to a PR in order to address review comments.

Reply:
done :)